draft-privacypass-rate-limit-tokens.md

```diff
@@ -736,7 +741,13 @@ Beyond the key configuration inputs, Clients also require the following inputs d
 in {{request-one}}: `token_key_id`, `blinded_msg`, `request_key`, and `name_key_id`.
 
 Together, these are used to encapsulate Origin Name (`origin_name`) and produce
-Encrypted Origin Name (`encrypted_origin`) as follows:
+Encrypted Origin Name (`encrypted_origin`).
+
+`origin_name` contains the name of the origin that initiated the challenge, as
+taken from the TokenChallenge.origin_info field. If the TokenChallenge.origin_info field
+is empty, `origin_name` is set to a fixed string "EmptyOrigin".
```

Comment:
```suggestion
is empty, `origin_name` is set to the empty string "".
```

Setting it to the empty string seems simplest here?

Reply:
We probably also need text in the Issuer->Attester section, or elsewhere, describing what to do when this value is received. Maybe we need a section specifically on how issuers should manage or configure origin rate limits?

Reply:
Yeah I was thinking about adding text for what to do when you receive an empty string. I'll add that.

Reply:
Added some text for this!